addons/cargo/XEH_postInit.sqf

```diff
@@ -36,6 +36,15 @@
     };
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(unloadedCargoOnKilled), {
```

Comment:
```suggestion
[QGVAR(unloadCargoOnKilled), {
```
Updated name makes more sense to me.

Reply:
`unloadCargoX` makes it sounds like it's doing the unloading to me, while `unloadedCargoX` makes it clearer that it happens after the cargo is unloaded.